README.md

```diff
@@ -209,44 +209,45 @@ oc apply -f deploy/olm-catalog/subscription.yaml
 ```
 
 ### Creating a Windows MachineSet
-Below is the example of a Windows MachineSet which can create Windows Machines that the WMCO can react upon. Please 
+Below is the example of an AWS Windows MachineSet which can create Windows Machines that the WMCO can react upon. Please
```

Comment:
In the future we may want to include example machinesets for each cloud provider. If we do do this it would be nice to move them into a separate example folder with different files for each cloud provider. We could link to the folder from the README to reduce the amount of sections that will be skipped over by users. 

Just something to think about, not a necessary change.

Reply:
Yes we can do that with each cloud provider we test and add. However I would like to leave the AWS example here.